app/assets/stylesheets/default.scss

```diff
@@ -14,4 +14,13 @@
     $card-background: #FF6201;
     border: 10px, solid, #FF6201;
     border-radius: 50%;
+    margin-bottom: 1rem;
+}
+
+.card {
+    max-width: 70%;
+    margin: auto;
+    margin-top: 1em;
+    position: relative;
```

Comment:
indentation in this file looks to be tab size 4,  this should be consistent throughout the project, other files look like they're set to `2` and should also be consistent between teammates!